### DISCUSSION TITLE
如何查询合并转发的群号

### DISCUSSION THREAD

**Discussion by ExHix** [Mrs4s/go-cqhttp#1498] [Q&A] [answered]
在nb2中使用`get_forward_msg(message_id=forward_id)`所得到的json只能得到发送者ID以及内容，如何得到这个合并转发原来的群号？

**Comment by synodriver** [comment]
等我加这个feature，这得动[miraigo](https://github.com/Mrs4s/MiraiGo/blob/master/client/multimsg.go#L145-L172)，要加兼容性测试

**Comment by synodriver** [comment]
https://github.com/Mrs4s/go-cqhttp/blob/54dbccf63cd9a7354144386598a999b842ed213e/coolq/api.go#L1534-L1541

**Comment by synodriver** [comment, marked as answer]
看最新版 已经有了
